package.json

```diff
@@ -31,8 +31,8 @@
     "immutable": "^3.8.1"
   },
   "peerDependencies": {
-    "react": "0.14.x || 15.x.x",
-    "react-dom": "0.14.x || 15.x.x"
+    "react": ">=0.14.x",
+    "react-dom": ">=0.14.x"
```

Comment:
Would probably be better to add || 16.x.x

Reply:
@benjy can this be merged now? we'd love to move over to react16 with react-rte in production :-)